modules/ssl/ssl_engine_vars.c

```diff
@@ -71,6 +76,21 @@ static int ssl_is_https(conn_rec *c)
     return sslconn && sslconn->ssl;
 }
 
+#if APR_VERSION_AT_LEAST(1,7,0)
```

Comment:
Is this really available in 1.7.0+ only?

Reply:
This API was.   But there are three to choose from, I switched to using the OpenSSL API instead int 531195b - thanks for reviewing!